src/components/common/UIInput/UIInput.stories.ts

```diff
@@ -0,0 +1,12 @@
+import type { Meta, StoryFn } from '@storybook/vue3'
+import UIInput from './UIInput.vue'
```

Comment:
```suggestion
import UiInput from './UiInput.vue'
```
Ui первое слово Input второе